### ISSUE TITLE
Required + nullable seems to not work

### ISSUE THREAD

**Issue by RoXuS** [acacode/swagger-typescript-api#144]
We have an issue if we use required AND nullable true on a property.

Our spec

		"OrganizationDto": {
			"type": "object",
			"required": ["fax"],
			"properties": {
			         ...
				"enabled": {
					"type": "boolean"
				},
				"fax": {
					"type": "string",
					"nullable": true
				},
                          ...
it produces

```
export interface OrganizationDto {
  enabled?: boolean;
  fax?: string | null;
```
instead of

```
export interface OrganizationDto {
  enabled?: boolean;
  fax: string | null;				
```

For us, undefined should be != than null.

If nullable is not set the property is ok fax: string.
If required is not set the property is ok too fax?: string | null

**Comment by RoXuS** [comment]
ok we was on swagger 2.0, no issue on openapi 3.0...

thx for your code.

**Comment by js2me** [comment]
Hello @RoXuS !   
"required" keyword should work with Swagger 2.0 spec.   

Here is example with "required"   
Schema:  
https://github.com/acacode/swagger-typescript-api/blob/master/tests/schemas/v2.0/example1.json#L278  
Generated:  
https://github.com/acacode/swagger-typescript-api/blob/master/tests/generated/v2.0/example1.ts#L30  


Which version of swagger-typescript-api are you using ?

**Comment by js2me** [comment]
@RoXuS can you share swagger schema?

**Comment by RoXuS** [comment]
yeah but we wanted use nullable but it seems not available on swagger 2.0...
We are migrating on openapi 3.0. Your library works as expected with openapi 3.0 + nullable + required.

Thx for your response and for your lib.
